src/components/SearchComponent.vue

```diff
@@ -22,17 +21,33 @@ defineProps({
   id: {
     type: String,
     required: true
+  },
+  dropdown: {
+    type: Array,
+    default: () => []
+  },
+  searchNamespace: {
```

Comment:
```suggestion
  searchEndpoint: {
```

Vu qu'on passe une URL.